app/models/course.rb

```diff
@@ -56,6 +56,7 @@ class Course < ApplicationRecord
   }.freeze
 
   DEGREE_REQUIREMENTS_REQUIRED_FROM = 2022
+  COURSE_2022_REQUIREMENTS_REQUIRED_FROM = 2022
```

Comment:
I don't think the name of the constant makes any sense. Don't worry, I tired something similar and failed to come up with something meaningful. In the end, I just use the integer 2022 directly.

Reply:
Reverted changes to just the integer 👍